dapos/dapos_transport_http.go

```diff
@@ -106,27 +121,39 @@ func (this *DAPoSService) getTransactionsByToAddressHandler(responseWriter http.
 	responseWriter.Write([]byte(response.String()))
 }
 
-// getTransactionsHandler
-func (this *DAPoSService) getTransactionsHandler(responseWriter http.ResponseWriter, request *http.Request) {
-	response := this.GetTransactions()
+// getTransactionsByFromAddressHandler
+func (this *DAPoSService) getTransactionsOldHandler(responseWriter http.ResponseWriter, request *http.Request) {
```

Comment:
We don't need to keep these old handlers around; deprecation is about functionality - not code. So long as we maintain the current functionality in the new handlers, we're fine.

Reply:
They are a little different, A: old one grabs ALL transactions, new one only grabs a few at a time. B. the way to call them is different too.